### ISSUE TITLE
for the generator +AnIn should accept +Inan or +Anim

### ISSUE THREAD

**Issue by reynoldsnlp** [reynoldsnlp/udar#16]
Generating a form that has `+AnIn` should work if you give it `+Inan` or `+Anim`.

Current behavior:
```
$ echo который+Pron+Rel+Neu+Inan+Sg+Acc | hdrus
который+Pron+Rel+Neu+Inan+Sg+Acc	который+Pron+Rel+Neu+Inan+Sg+Acc+?	inf
$ echo который+Pron+Rel+Neu+AnIn+Sg+Acc | hdrus
который+Pron+Rel+Neu+AnIn+Sg+Acc	которое	6.521484
```

Desired behavior:

```
$ echo который+Pron+Rel+Neu+Inan+Sg+Acc | hdrus
который+Pron+Rel+Neu+Inan+Sg+Acc	которое	6.521484
```

**Comment by reynoldsnlp** [comment]
This is fixed in the `giellatekno` repo, just need to push the compiled transducers here.

**Comment by reynoldsnlp** [comment]
Fixed in 0fd19669f73c311e94990dfaaf60268b6dd49e2e
